ext/soap/php_sdl.c

```diff
@@ -922,14 +920,14 @@ static sdlPtr load_wsdl(zval *this_ptr, char *struri)
 
 						soapOperation = get_node_ex(operation->children, "operation", wsdl_soap_namespace);
 						if (soapOperation) {
-							tmp = get_attribute(soapOperation->properties, "soapAction");
-							if (tmp) {
-								soapFunctionBinding->soapAction = estrdup((char*)tmp->children->content);
+							xmlAttrPtr style_soap_action_attribute = get_attribute(soapOperation->properties, "soapAction");
+							if (style_soap_action_attribute) {
+								soapFunctionBinding->soapAction = estrdup((char*)style_soap_action_attribute->children->content);
 							}
 
-							tmp = get_attribute(soapOperation->properties, "style");
-							if (tmp) {
-								if (!strncmp((char*)tmp->children->content, "rpc", sizeof("rpc"))) {
+							xmlAttrPtr style_attribute = get_attribute(soapOperation->properties, "style");
```

Comment:
Ah now I know where "style" comes from :P, but the previous variable rename is not related to style...
Same camelCase comment